README.md

```diff
@@ -331,6 +331,8 @@ $ amtool config routes test --config.file=doc/examples/simple.yml --tree --verif
 Alertmanager's high availability is in production use at many companies and is enabled by default.
 
 > Important: Both UDP and TCP are needed in alertmanager 0.15 and higher for the cluster to work.
+>  - If you are using a firewall, make sure to whitelist the clustering port for both protocols.
+>  - If you are using docker, make sure to expose the clustering port for both protocols (i.e. `-p 9094:9094/tcp -p 9094:9094/udp`).
```

Comment:
"if you are using docker" -> "if you are running in a container"

Reply:
Containers don't have command line flags.

Reply:
Hence removing the (i.e. ...) part.

Reply:
Changed.